twib/Logger.hpp

```diff
@@ -10,13 +55,13 @@ enum class Level {
 	INFO,
 	MSG,
 	WARN,
-	ERROR,
+	ERR,
 	FATAL,
 	MAX
 };
 
 #define log(lvl, format, ...) \
-	_log(::twili::log::Level::lvl, __FILE_SHORT__, __LINE__,	\
+	_log(::twili::log::Level::lvl, __FILE__, __LINE__,	\
```

Comment:
Does the `__FILE_SHORT__` cmake foolery not work on Windows? I'd rather either use a macro here to use `__FILE__` on Windows and `__FILE_SHORT__` on other platforms, or change the cmake file to provide `__FILE_SHORT__` (working or not) on Windows. I don't really like having unnecessarily long prologues in my logs.

Reply:
__FILE_SHORT__ returned an int on windows instead of a char*, which would obviously not work.

Reply:
Hmm, that doesn't make any sense. `FILE_SHORT` comes from our CMake config